source/features/extend-diff-expander.tsx

```diff
@@ -25,5 +21,7 @@ features.add({
 		pageDetect.isPRFiles,
 		pageDetect.isCommit
 	],
-	init
+	init: () => {
+		delegate(document, '.diff-view .js-expandable-line', 'click', init);
+	}
```

Comment:
What do you say to init's that have just a simple delegate calling another function? Move them here or leave them alone?

Reply:
I'm conflicted on this one. We do have other "events" in the `features.add` object, so maybe it makes sense to also have `delegate` here.

But maybe not: `delegate` is used for user events (`click`, etc) so it's part of the feature rather than the loading logic.

You can revert this.